src/string.rs

```diff
@@ -151,3 +151,21 @@ impl Deref for String {
     unsafe { &*(self as *const _ as *const Value) }
   }
 }
+
+impl Into<String> for Value {
+  fn into(self) -> String {
+    // TODO: do this check in debug mode only
+    if !self.is_string() {
+      panic!("unable to convert")
+    } 
+    
+    unsafe { std::mem::transmute(self) }
+  }
+}
+
+impl<'sc> Into<Local<'sc, String>> for Local<'sc, Value>
+{
+  fn into(self) -> Local<'sc, String> {
+    unsafe { std::mem::transmute(self) }
+  }
+}
```

Comment:
This is interesting; in `local.rs` we have:
```rust
impl<'sc, T> From<Local<'sc, T>> for Local<'sc, Value>
where
  T: Deref<Target = Value>,
{
  fn from(v: Local<'sc, T>) -> Local<'sc, Value> {
    unsafe { std::mem::transmute(v) }
  }
}
```
If type implements `From` then blanket implementation for `Into` is provided, but in the case above it's not working without explicit implementation.

Reply:
I also tried something similar, the compiler prompts conflict